src/cli.rs

```diff
@@ -626,10 +626,10 @@ fn init_commands<'a, C: CLI<'a>>() -> Vec<Command<'a, C>> {
                                 }
                             };
                             // Try to get a readable name for the asset.
-                            let asset = if txn.asset == AssetCode::native() {
+                            let asset = if *txn.asset() == AssetCode::native() {
```

Comment:
Idea for cleaning this sort of thing up a little: https://github.com/EspressoSystems/seahorse/issues/243